.gitignore

```diff
@@ -1 +1,7 @@
 *.pyc
+.vscode/
+bin/
+include/
+lib/
+.Python
+
```

Comment:
Additional newline.

Reply:
Sorry, I don't follow. Should I kill line 7 or add/modify something instead?

Reply:
Solved, turn out my vscode automatically add additional LF for me.